response_operations_ui/views/surveys.py

```diff
@@ -46,3 +53,27 @@ def view_survey(short_name):
                            survey=survey_details['survey'],
                            collection_exercises=survey_details['collection_exercises'],
                            breadcrumbs=breadcrumbs)
+
+
+@surveys_bp.route('/edit-survey-details/<short_name>', methods=['GET'])
+@login_required
+def view_survey_details(short_name):
+    survey_details = survey_controllers.get_survey(short_name)
+    form = EditSurveyDetailsForm(form=request.form)
+
+    return render_template('edit-survey-details.html', form=form, short_name=short_name,
+                           legal_basis=survey_details['survey']['legalBasis'],
+                           long_name=survey_details['survey']['longName'],
+                           survey_ref=survey_details['survey']['surveyRef'])
+
+
+@surveys_bp.route('/edit-survey-details/<short_name>', methods=['POST'])
+@login_required
+def edit_survey_details(short_name):
+    form = request.form
```

Comment:
Are we not doing any validation on this form? e.g. we probably shouldn't be able to send letters for the surveyref

Reply:
Would this be fixed if I just passed `survey_ref` through the url instead of having a hidden field in the form then I wouldn't have the need to validate form?

Reply:
I think the best way would be to add validation in the flaskform class